### PR TITLE
fix(OpenAPI): Fix array syntaxes

### PR DESCRIPTION
**Pull request by provokateurin**
For https://github.com/nextcloud/openapi-extractor/issues/33 and tested with https://github.com/nextcloud/openapi-extractor/pull/168

### CODE REVIEW

File: lib/Controller/EndpointController.php [hunk 1]
```diff
@@ -195,10 +195,10 @@ public function confirmIdsForUser(string $apiVersion, array $ids): DataResponse
 			return new DataResponse([], Http::STATUS_BAD_REQUEST);
 		}
 
-		$ids = array_unique(array_filter(array_map(
+		$ids = array_values(array_unique(array_filter(array_map(
```
Comment (nickvergessen) [inline lib/Controller/EndpointController.php:198]: This is not what we return, so I don't really get why this is needed (just for understanding it)

File: lib/Handler.php [hunk 2]
```diff
@@ -183,8 +183,8 @@ public function getById(int $id, string $user): INotification {
 	/**
 	 * Confirm that the notification ids still exist for the user
 	 *
-	 * @param int[] $ids
-	 * @return int[]
+	 * @param list<int> $ids
```
Comment (nickvergessen) [inline lib/Handler.php:186]: I guess because you changed this here which is also technically not needed?

**Comment by provokateurin** [comment]
Sorry I accidentally deleted the comment, I didn't mean to do so.

File: lib/Controller/EndpointController.php (hunk 1 above)
Comment (provokateurin) [reply lib/Controller/EndpointController.php:198]: Yes, but it is cleaner to make the `confirmIdsForUser()` also only accept and return `list<int>`.
Technically not needed to change this, but I did it for completeness.

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**